scripts/promptlang.py

```diff
@@ -107,5 +106,6 @@ def title(self):
     def show(self, is_img2img):
         return scripts.AlwaysVisible
 
-    def run(self, p, *args):
-        pass
+    def process(self, p, *args):
+        global_state.is_negative = True
```

Comment:
monkey patch multicond, set to true before original multicond, and set to false at the end of simplecond above.

Reply:
because here it does not scale for extensions that call get learned cond multiple times